src/PROPOSAL/detail/PROPOSAL/Constants.cxx

```diff
@@ -30,6 +30,7 @@ unsigned int InterpolationSettings::NODES_DEDX = 500;
 unsigned int InterpolationSettings::NODES_DE2DX = 200;
 unsigned int InterpolationSettings::NODES_DNDX_E = 100;
 unsigned int InterpolationSettings::NODES_DNDX_V = 100;
+unsigned int InterpolationSettings::NODES_UTILITY = 500;
```

Comment:
I am surprised, that the number of interpolation nodes for the propagation integrals is 500.
In the old PROPOSAL this value was 1000, see e.g. in version 3
https://github.com/tudo-astroparticlephysics/PROPOSAL/blob/v3.0.0/public/PROPOSAL/Constants.h#L62
which was for example used in 
https://github.com/tudo-astroparticlephysics/PROPOSAL/blob/v3.0.0/private/PROPOSAL/ProcessCollection.cxx#L587

When did this change?

Reply:
in version 5.2.0 it was still 1000, see
https://github.com/tudo-astroparticlephysics/PROPOSAL/blob/5.2.0/public/PROPOSAL/methods.h#L108

Reply:
I am not sure why this number changed. In principal, it would be no problem to go back to 1000 nodes. The size of the propagation utility tables is only around 4 KB for 500 nodes, which is negligible compared to to dNdx tables which have a size of around 320 KB.

Reply:
Yes, setting the number of nodes of 1000 does not change the runtime, as expected

Reply:
how much does this affect the accuracy?
I remember that 100 were the maximum for cross sections since they are two dimensional.
For Continuous Randomization, 100 was not accurate enough, therefore it was set to 200.
However, for the propagation integrals, a high accuracy was targeted. But maybe 500 nodes are already sufficient..
Can you quantify the decrease in accuracy and the gain in runtime when changing the nodes from 1000 to 500?

Reply:
But the question, to change the number of nodes back to 1000 should be tested in another PR. This one is fine, so far.